volontulo/tests/views/test_auth.py

```diff
@@ -125,3 +125,131 @@ def test__logged_out_authenticated_user(self):
             u"Użytkownik został wylogowany!"
         )
         self.assertNotIn('_auth_user_id', self.client.session)
+
```

Comment:
Right now we have separate classes for every view - that should stay like that, so You need to create separate classes for TestLogin and TestLogout.